src/interface_adapter/add_project/AddProjectPresenter

```diff
@@ -0,0 +1,31 @@
+package interface_adapter.dashboard;
+
+import use_case.add_project.AddProjectOutputBoundary;
+import use_case.add_project.AddProjectOutputData;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class AddProjectPresenter implements AddProjectOutputBoundary {
+
+    private final DashboardViewModel dashboardViewModel;
+
+    public AddProjectPresenter(DashboardViewModel dashboardViewModel) {
+        this.dashboardViewModel = dashboardViewModel;
+    }
+
+    @Override
+    public void prepareSuccessView(AddProjectOutputData response) {
+        DashboardState dashboardState = dashboardViewModel.getState();
+        dashboardState.addProjectData(response);
+
+        addProjectViewModel.firePropertyChanged();
```

Comment:
ISSUE: addProjectViewModel does not exist in this file. Do you mean dashboardViewModel? (also at line 29)

Reply:
addProjectViewModel was implemented before dashboardViewModel was created and was promptly removed afterwards. I'm not exactly sure how you found the file but it's not in the Pull Request.